ci/scripts/python_wheel_manylinux_build.sh

```diff
@@ -53,6 +53,7 @@ echo "=== (${PYTHON_VERSION}) Building Arrow C++ libraries ==="
 : ${ARROW_FLIGHT:=ON}
 : ${ARROW_GANDIVA:=OFF}
 : ${ARROW_GCS:=ON}
+: ${ARROW_AZURE:=ON}
```

Comment:
Could you keep this list in alphabetical order?
Could you use `OFF` as the default value for now because we don't have pyarrow bindings yet?

Reply:
I have fixed the ordering and changed it to OFF. Personally though I would have prefered to have it ON to make sure the Azure SDK build is working and covered by CI in this scenario.

Reply:
OK. Then we can use `ON`.